modules/VTK/wrapping/CMakeLists.txt

```diff
@@ -15,26 +17,47 @@ link_directories(
   ${HDF5_LIBRARY_DIR}
 )
 
+# The process of wrapping requires some numpy headers.
+# (Not entirely sure what requires it though)
+if( True )
```

Comment:
I guess this should be ``` if( ${BUILD_WRAPPING} )```

Reply:
Not actually, because we are already in the wrapping sub-folder, that is only included if if( ${BUILD_WRAPPING} ) true is. Best is to remove the if ( True ). I figured out by now that the numpy dependency is hard (numpy.i requires numpy/arrayobject.h)